superbit_lensing/medsmaker/scripts/process_mocks.py

```diff
@@ -60,26 +63,25 @@ def main():
     else:
         fname_base = args.fname_base
 
+
     if (select_truth_stars=='true') or (select_truth_stars=='True'):
         select_truth_stars = True
     else:
         select_truth_stars = False
 
+
     science = glob.glob(os.path.join(mock_dir, fname_base)+'*[!truth,mcal,.sub].fits')
     logprint(f'Science frames: {science}')
 
-
     outfile = os.path.join(outdir, outfile)
 
     logprint('Setting up configuration...')
     bm = medsmaker.BITMeasurement(image_files=science, data_dir=mock_dir, log=log, vb=vb)
 
     bm.set_working_dir(path=outdir)
     bm.set_path_to_psf(path=os.path.join(outdir, 'psfex_output'))
-
-    # Make a mask.
-    logprint('Making mask...')
-    bm.make_mask(mask_name='forecast_mask.fits')
+    bm.set_mask(mask_name='forecast_mask.fits',mask_dir=os.path.join(mock_dir,'mask_files'))
+    bm.set_weight(weight_name='forecast_weight.fits',weight_dir=os.path.join(mock_dir,'weight_files'))
```

Comment:
maybe it's just an issue w/ the rendering, but these two new lines look like they are not indexed correctly